src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/DataUtils.java

```diff
@@ -0,0 +1,64 @@
+package com.github.markusbernhardt.seleniumlibrary.keywords;
+
+import com.github.markusbernhardt.seleniumlibrary.RunOnFailureKeywordsAdapter;
+import org.robotframework.javalib.annotation.ArgumentNames;
+import org.robotframework.javalib.annotation.Autowired;
+import org.robotframework.javalib.annotation.RobotKeyword;
+import org.robotframework.javalib.annotation.RobotKeywords;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@RobotKeywords
+public class DataUtils extends RunOnFailureKeywordsAdapter {
+
+    private static final String ASCENDING = "ascending";
+    private static final String DESCENDING = "descending";
+
+    @Autowired
+    protected Robot robot;
+
+    /**
+     * Instantiated Logging keyword bean
+     */
+    @Autowired
+    protected Logging logging;
+
+    // ##############################
+    // Keywords
+    // ##############################
+
+    @RobotKeyword
```

Comment:
If we add a new keywords, should we add documentation to them as well? Ideally, to also have tests ;-)

Reply:
Done